### ISSUE TITLE
Problems installing BioSequences package

### ISSUE THREAD

**Issue by azdaly** [BioJulia/BioSequences.jl#71]
Hello! Apologies if this is not the right place to post things like this. I am trying to install the BioSequences package, and have done the following:
* Entered package mode    
```registry add https://github.com/BioJulia/BioJuliaRegistry.git```    
* Exited package mode    
```using Pkg```    
```Pkg.add("BioSequences")```

I am getting the following error:    

```
ERROR: MethodError: Cannot `convert` an object of type Nothing to an object of type String    
Closest candidates are:     
  convert(::Type{T<:AbstractString}, ::T<:AbstractString) where T<:AbstractString at strings/basic.jl:208    
  convert(::Type{T<:AbstractString}, ::AbstractString) where T<:AbstractString at strings/basic.jl:209     
  convert(::Type{T}, ::T) where T at essentials.jl:167     
Stacktrace:     
 [1] setindex!(::Dict{Base.UUID,String}, ::Nothing, ::Base.UUID) at ./dict.jl:380    
 [2] deps_graph(::Pkg.Types.Context, ::Dict{Base.UUID,String}, ::Dict{Base.UUID,Pkg.Types.VersionSpec}, ::Dict{Base.UUID,Pkg.Types.Fixed}) at /Users/sabae/buildbot/worker/package_macos64/build/usr/share/julia/stdlib/v1.2/Pkg/src/Operations.jl:424    
 [3] resolve_versions!(::Pkg.Types.Context, ::Array{Pkg.Types.PackageSpec,1}) at /Users/sabae/buildbot/worker/package_macos64/build/usr/share/julia/stdlib/v1.2/Pkg/src/Operations.jl:316    
 [4] #add#100(::Bool, ::typeof(Pkg.Operations.add), ::Pkg.Types.Context, ::Array{Pkg.Types.PackageSpec,1}, ::Array{Base.UUID,1}) at /Users/sabae/buildbot/worker/package_macos64/build/usr/share/julia/stdlib/v1.2/Pkg/src/Operations.jl:962    
 [5] #add at ./none:0 [inlined]    
 [6] #add#25(::Bool, ::Base.Iterators.Pairs{Union{},Union{},Tuple{},NamedTuple{(),Tuple{}}}, ::typeof(Pkg.API.add), ::Pkg.Types.Context, ::Array{Pkg.Types.PackageSpec,1}) at /Users/sabae/buildbot/worker/package_macos64/build/usr/share/julia/stdlib/v1.2/Pkg/src/API.jl:99    
 [7] add at /Users/sabae/buildbot/worker/package_macos64/build/usr/share/julia/stdlib/v1.2/Pkg/src/API.jl:69 [inlined]    
 [8] #add#24 at /Users/sabae/buildbot/worker/package_macos64/build/usr/share/julia/stdlib/v1.2/Pkg/src/API.jl:67 [inlined]    
 [9] add at /Users/sabae/buildbot/worker/package_macos64/build/usr/share/julia/stdlib/v1.2/Pkg/src/API.jl:67 [inlined]     
 [10] #add#21 at /Users/sabae/buildbot/worker/package_macos64/build/usr/share/julia/stdlib/v1.2/Pkg/src/API.jl:65 [inlined]    
 [11] add at /Users/sabae/buildbot/worker/package_macos64/build/usr/share/julia/stdlib/v1.2/Pkg/src/API.jl:65 [inlined]    
 [12] #add#20(::Base.Iterators.Pairs{Union{},Union{},Tuple{},NamedTuple{(),Tuple{}}}, ::typeof(Pkg.API.add), ::String) at /Users/sabae/buildbot/worker/package_macos64/build/usr/share/julia/stdlib/v1.2/Pkg/src/API.jl:64    
 [13] add(::String) at /Users/sabae/buildbot/worker/package_macos64/build/usr/share/julia/stdlib/v1.2/Pkg/src/API.jl:64    
 [14] top-level scope at REPL[2]:1    
```    
    
I'm using Julia version 1.2.0, and the latest BioSequences. I'm fairly confident that I got the git repo correctly, because re-doing:
```registry add https://github.com/BioJulia/BioJuliaRegistry.git```
Results in:
```
   Cloning registry from "https://github.com/BioJulia/BioJuliaRegistry.git"    
[ Info: registry `BioJuliaRegistry` already exist in `~/.julia/registries/BioJuliaRegistry`. 
```

**Comment by BenJWard** [comment]
BioSequences is currently not tested on 1.2, but I'm going to enable it #72. However, it looks like from your log that it's not to do with BioSequences, because the error looks like it is comming from base julia. I'll install 1.2 later myself today and see if it happens to me.

**Comment by BenJWard** [comment]
I think BioSequences was released at v2 before 1.2 existed.

I just compiled a fresh julia v1.2 on my machine and I can't reproduce the issue:

```julia
N82106:julia1_2 bward$ ./julia
               _
   _       _ _(_)_     |  Documentation: https://docs.julialang.org
  (_)     | (_) (_)    |
   _ _   _| |_  __ _   |  Type "?" for help, "]?" for Pkg help.
  | | | | | | |/ _` |  |
  | | |_| | | | (_| |  |  Version 1.2.0 (2019-08-20)
 _/ |\__'_|_|_|\__'_|  |  
|__/                   |

(v1.2) pkg> registry add https://github.com/BioJulia/BioJuliaRegistry.git
   Cloning registry from "https://github.com/BioJulia/BioJuliaRegistry.git"
[ Info: registry `BioJuliaRegistry` already exist in `~/.julia/registries/BioJuliaRegistry`.

(v1.2) pkg> add BioSequences
  Updating registry at `~/.julia/registries/BioJuliaRegistry`
  Updating git-repo `https://github.com/BioJulia/BioJuliaRegistry.git`
  Updating registry at `~/.julia/registries/General`
  Updating git-repo `https://github.com/JuliaRegistries/General.git`
 Resolving package versions...
  Updating `~/.julia/environments/v1.2/Project.toml`
  [7e6ae17a] + BioSequences v2.0.0
  Updating `~/.julia/environments/v1.2/Manifest.toml`
  [67c07d97] + Automa v0.7.0
  [47718e42] + BioGenerics v0.1.0
  [7e6ae17a] + BioSequences v2.0.0
  [3c28c6f8] + BioSymbols v4.0.1
  [861a8166] + Combinatorics v0.7.0
  [864edb3b] + DataStructures v0.17.0
  [1cb3b9ac] + IndexableBitVectors v1.0.0
  [bac558e1] + OrderedCollections v1.1.0
  [f27b6e38] + Polynomials v0.5.2
  [3bb67fe8] + TranscodingStreams v0.9.5
  [7200193e] + Twiddle v1.1.1
  [2a0f44e3] + Base64 
  [8ba89e20] + Distributed 
  [b77e0a4c] + InteractiveUtils 
  [8f399da3] + Libdl 
  [37e2e46d] + LinearAlgebra 
  [56ddb016] + Logging 
  [d6f4376e] + Markdown 
  [de0858da] + Printf 
  [9a3f8284] + Random 
  [9e88b42a] + Serialization 
  [6462fe0b] + Sockets 
  [2f01184e] + SparseArrays 
  [8dfed614] + Test 
  [4ec0a83e] + Unicode 

(v1.2) pkg> 
```
I wasn't talking about Project.toml, I was talking about travis.

The Project.toml is fine as far as I can tell: In that context, it means any julia 1.X version until julia 2.0, that is higher than 1.1, which to me was a reasonable specification. And if it was wrong, you wouldn't get the error demonstrated here, you'd get an error message about being unable to resolve versions.

**Comment by jakobnissen** [comment]
This has been marked as "can't replicate" for 1 year+. @azdaly closing this. If you didn't resolve it, please make another issue.
